index.html

```diff
@@ -9,84 +9,63 @@
     <link
       rel="stylesheet"
       href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css"
     />
     <title>Portfolio</title>
 </head>
 <body>
-    <section>
-        <nav id="navbar" class="navbar fixed-top navbar-expand-lg navbar-dark bg-secondary">
-          <div class="container-fluid">
-            <a class="navbar-brand" href="#">John Doe</a>
-            <button class="navbar-toggler" 
-              type="button" 
-              data-bs-toggle="collapse" 
-              data-bs-target="#navbarNavAltMarkup"
-              aria-controls="navbarNavAltMarkup"
-              aria-expanded="false"
-              aria-label="Toggle navigation"
-            >
-              <span class="navbar-toggler-icon"></span>
-            </button>
-            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
-              <div class="navbar-nav">
-                <a class="nav-link text-white" href="#portfolio">Portfolio</a>
-                <a class="nav-link text-white" href="#about">About</a>
-                <a class="nav-link text-white" href="#contact">Contact</a>
-                <i class="bi bi-envelope text-white"></i>
-              </div>
-            </div>
-          </div>
-        </nav>
-    </section>
+    <header class="top-nav">
+      <div class="topnav">
+        <a href="#home" class="active">John Doe</a>
+        <div class="my-link">
+        </div>
+        <a href="javascript:void(0);" class="icon">
+          <i class="fa fa-bars"></i>
+        </a>
+      </div>
+    </header>
       <!-- HOME SECTION -->
-      <section class="container-fluid bg-dark pt-5 pt-lg-0">
+      <section class="container-fluid bg-dark">
         <div class="row py-5 py-lg-2">
-          <div class="col-lg-1 d-flex align-items-center order-3 order-lg-0">
-            <ul class="tags flex-lg-column flex-md-row">
-              <li class="elements">
-                <a href="#">
-                  <i class="bi bi-twitter text-white"></i>
-                </a>
-              </li>
-              <li class="mb-2 me-2">
-                <a href="#">
-                  <i class="bi bi-linkedin text-white"></i>
-                </a>
-              </li>
-              <li class="mb-2 me-2">
-                <a href="#">
-                  <i class="bi bi-instagram text-white"></i>
-                </a>
-              </li>
-              <li class="mb-2 me-2">
-                <a href="#">
-                  <i class="bi bi-github text-white"></i>
-                </a>
-              </li>
-            </ul>
-          </div>
-          <div class="col-lg-5 text-white d-flex flex-column justify-content-center">
-            <h1 class="text-danger h1 mb-3">
-              Hey there.<br />
+          <div>
+            <h1 class="heading">
+              Hey There.<br />
               I’m Alex
             </h1>
```

Comment:
- [ ] As the previous reviewer mentioned, it's better to remove the `br` tag as it is better to use CSS to fix and implement that, kindly change that. 😁😁